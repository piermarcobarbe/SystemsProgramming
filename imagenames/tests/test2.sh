#!/bin/sh

exec "$1" "tests/extensions1" <<EOF
android/Messaging1550748367499.jpg
books/feyerabend_paul_against_method.pdf
gpu_computing_gems_9780123849892.pdf
strange_fruit.html
android/Messaging1552068751642.jpg
android/smile.gif
lib/images/usi-logo.png
lib/images/usi-logo.pdf
weather/lugano-2019.04.03.png
beyond_vietnam.html
lib/images/pantheon.jpeg
lib/images/ANTO.JPEG
EOF
