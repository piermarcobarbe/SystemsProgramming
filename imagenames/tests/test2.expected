Messaging1550748367499
Messaging1552068751642
pantheon
ANTO
